db/migrations/0023.relax_course_name.py

```diff
@@ -0,0 +1,16 @@
+#
+# file: migrations/0023.relax_course_name.py
+#
+from yoyo import step
+
+steps = [
+    step('''
+        ALTER TABLE course
+        MODIFY
+            name VARCHAR(256) NOT NULL;
+    '''),
+    step('''
+        ALTER TABLE course
```

Comment:
Do you need to drop the index first before increasing the size?

I believe that on utf8mb4 the max indexable field size is 191 unless you we have the large prefix option.

We could still index first 191 characters though. But maybe an index isn't needed at all for this field really.

Reply:
Um, I'm dropping the index all together, right? I can re-order them if you think it might be a problem. It wasn't locally using `docker-compose`.

Reply:
Yeah I just didn't know if it would cause an error if you had a value that was too big for the index. I didn't actually test this. If you drop it first it for sure wouldn't be an issue.

It's possible MySQL might just keep the index but keep it at a smaller length until it was dropped.

Reply:
I re-ordered them. Please take a look.

Reply:
Yup looks fine.

Since these both reference the same table you *could* have just done it in the same `ALTER`. I found out about this in my other PR, I actually wasn't 100% sure. But this is just a nitpick. 

```
    step('''

        ALTER TABLE course
        DROP INDEX name, 
        MODIFY
            name VARCHAR(256) NOT NULL;
    ''')

```